### PR TITLE
add objectives to sql

### PR DESCRIPTION
**Pull request by lizTheDeveloper**
Fixes part of #633

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/add-column.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/add-constraint.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/add-default-value.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/add-rule.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/alter/add-trigger.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/create-a-database.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/create/create-a-table.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/add-support-for-other-character-sets.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/encoding-and-collation.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-numeric-data-types.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-string-data-types.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/data-types/sql-text-data-type.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/create-indexes.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/indices/when-to-add-indexes.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/blobs-in-a-rdbms.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-mssql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-mysql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-blobs/storing-blobs-in-psql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-an-rdbms.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-mssql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-mysql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/ddl/storing-json/storing-json-in-postgresql.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/avg-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/count-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/distinct-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/group-by-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/min-and-max-clauses.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/aggregate-queries/sum-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/order-by-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/select.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-2.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where-3.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/basic-queries/where.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/command-line-client.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/gui-client.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/java-database-packages.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/node-database-packages.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/python-database-packages.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/database-clients/what-is-a-database-client.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/full-join.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/inner-join.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/left-and-right-joins.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/multiple-joins.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/joins/union.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/connection-strings.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/what-is-sql/database-server-types.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/delete-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/insert.md?ref=f753de05271f42191aee88c401b71cfe55044768
- [ ] https://enkipro.compreview-insight/enkidevs/curriculum/sql/dql/write/update-clause.md?ref=f753de05271f42191aee88c401b71cfe55044768

File: sql/ddl/alter/add-rule.md [hunk 1]
```diff
@@ -17,9 +17,6 @@ category: must-know
 
 inAlgoPool: false
 
-standards:
```
Comment (Loopiezlol) [inline sql/ddl/alter/add-rule.md]: was this on purpose?

File: sql/ddl/alter/add-default-value.md [hunk 2]
```diff
@@ -17,9 +17,6 @@ category: must-know
 
 inAlgoPool: false
 
-standards:
-  sql.define-sequences-or-default-values: 10
```
Comment (Loopiezlol) [inline sql/ddl/alter/add-default-value.md]: also this was on purpose too?

**Comment by lizTheDeveloper** [comment]
@Loopiezlol we're good to merge this

**Pull request merged**